text/0018-transaction-payload/0018-transaction-payload.md

```diff
@@ -460,45 +377,41 @@ Consider a <i>Transaction Essence</i> containing the <i>UTXO Inputs</i> 0, 1 and
 
 ## Validation
 
-A <i>Transaction Payload</i> has different validation stages, since some validation steps can only be executed when certain information has (or has not) been received. We therefore distinguish between syntactic and semantic validation:
+A <i>Transaction Payload</i> has different validation stages, since some validation steps can only be executed when certain information has (or has not) been received. We therefore distinguish between syntactic and semantic validation
+
+The different output types and optional output feature blocks introduced by [draft RFC-0038](https://github.com/iotaledger/protocol-rfcs/pull/38) add additional constraints to the transaction validation rules, but since these are specific to the given outputs and features, they are discussed for each [output type](https://github.com/lzpap/protocol-rfcs/blob/master/text/0038-output-types-for-tokenization-and-sc/0038-output-types-for-tokenization-and-sc.md#output-design) and [feature block type](https://github.com/lzpap/protocol-rfcs/blob/master/text/0038-output-types-for-tokenization-and-sc/0038-output-types-for-tokenization-and-sc.md#optional-output-features) separately.
```

Comment:
```suggestion
The different output types and optional output feature blocks introduced by [draft RFC-0038](https://github.com/iotaledger/protocol-rfcs/pull/38) add additional constraints to the transaction validation rules, but since these are specific to the given outputs and features, they are discussed for each [output type](https://github.com/lzpap/protocol-rfcs/blob/master/tips/TIP-0018/tip-0018.md#output-design) and [feature block type](https://github.com/lzpap/protocol-rfcs/blob/master/tips/TIP-0018/tip-0018.md#optional-output-features) separately.
```